README.Rmd

```diff
@@ -269,15 +288,15 @@ processed_data_plot <-
 processed_data_plot
 ```
 
-To make a forecast, we will use a "canned" simple auto-regressive forecaster to
+To make a forecast, we will use a simple “canned” auto-regressive forecaster to
 predict the death rate four weeks into the future using lagged[^3] deaths and
-cases
+cases.
 
 [^3]: lagged by 3 in this context meaning using the value from 3 days ago.
 
 ```{r make-forecasts, warning=FALSE}
 four_week_ahead <- arx_forecaster(
-  cases_deaths |> filter(time_value <= forecast_date),
+  covid_case_death_rates |> filter(time_value <= forecast_date),
```

Comment:
This ends up being inconsistent with the fold (we'll be fitting on the unsmoothed version etc). I went back and forth with Dan about eliminating that section and just using the raw data like this

Reply:
Actually, I forgot we had done this, but `covid_case_death_rates` is actually from the 7dav signal, so I should probably greatly curtail that note either way